src/py42/clients/securitydata.py

```diff
@@ -185,6 +185,25 @@ def search_file_events(self, query):
         """
         return self._file_event_service.search(query)
 
+    def get_all_file_events(self, query, page_token=""):
```

Comment:
We should consider renaming this to `search_for_all_file_events(query)` because all of the search methods in py42 take these query objects, and that maintains that consistency.  @alanag13

Reply:
agreed, also, since this doesn't use a generator we should not use `get_all`.

Reply:
Changed.